analysis/filter/EdgeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/edgefilter/EdgeFilter.kt

```diff
@@ -22,8 +24,8 @@ class EdgeFilter(
     @CommandLine.Option(names = ["-h", "--help"], usageHelp = true, description = ["displays this help and exits"])
     var help: Boolean = false
 
-    @CommandLine.Parameters(arity = "1..*", paramLabel = "FILE", description = ["files to filter"])
```

Comment:
I am not sure if this was the correct arity, since from my understanding we only ever wanted to use one file with the EdgeFilter.